tools/src/wyvern/tools/typedAST/core/binding/AliasBinding.java

```diff
@@ -0,0 +1,40 @@
+package wyvern.tools.typedAST.core.binding;
+
+import wyvern.tools.types.Type;
+import wyvern.tools.util.TreeWriter;
+
+public class AliasBinding implements Binding {
+
```

Comment:
This class is not used anywhere else in the pull request, so it should be deleted.